field/FieldGet.F90

```diff
@@ -0,0 +1,53 @@
+#include "MAPL_Generic.h"
+
+module mapl3g_FieldGet
+   use mapl3g_FieldInfo
+   use mapl_KeywordEnforcer
+   use mapl_ErrorHandling
+   use esmf
+   implicit none (type, external)
+   private
+
+   public :: MAPL_FieldGet
+
+   interface MAPL_FieldGet
+      procedure field_get
+   end interface MAPL_FieldGet
+
+contains
+
+!#   subroutine field_get (field, unusable, &
+!#        ! pass thru to ESMF
+!#        status, geomtype, geom, typekind,  rank, dimCount, staggerloc, name, vm, &
+!#        ! allocatable in MAPL
+!#        minIndex,  maxIndex, elementCount, &
+!#        localMinIndex, localMaxIndex, &
+!#        ! MAPL specific
+!#        units, standard_name, long_name, &
+!#        rc)
+!#
+!#   end subroutine field_get
```

Comment:
Will this block be uncommented and used eventually, or it is a remnant?

Reply:
I've deleted in my branch where I'm extending and integrating this.